### ISSUE TITLE
After Update to 0.4.10 error Cannot read property 'cookie' of undefined

### ISSUE THREAD

**Issue by DJay-X** [NorthernMan54/homebridge-alexa#209]
After Update to 0.4.10 error Cannot read property 'cookie' of undefined
Back to v0.4.9 and Homebridge is running again okay.
homebridge@0.4.48 | node@11.14.0 | npm 6.9.0 | n@3.0.2
Thx

```
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]: [2019-4-21 22:01:18] TypeError: Cannot read property 'cookie' of undefined
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at Accessory.toAlexa (/usr/local/lib/node_modules/homebridge-alexa/lib/parse/Accessory.js:92:44)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at Homebridge.toAlexa (/usr/local/lib/node_modules/homebridge-alexa/lib/parse/Homebridge.js:161:34)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at Homebridges.toAlexa (/usr/local/lib/node_modules/homebridge-alexa/lib/parse/Homebridges.js:47:35)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-alexa/lib/alexaActions.js:98:28)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at HAPNodeJSClient.HAPaccessories (/usr/local/lib/node_modules/homebridge-alexa/node_modules/hap-node-client/HAPNodeJSClient.js:156:3)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at Object.alexaDiscovery (/usr/local/lib/node_modules/homebridge-alexa/lib/alexaActions.js:91:14)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at HAPNodeJSClient.<anonymous> (/usr/local/lib/node_modules/homebridge-alexa/lib/alexaActions.js:31:20)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at HAPNodeJSClient.emit (events.js:193:13)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at HAPNodeJSClient._discoveryEnd (/usr/local/lib/node_modules/homebridge-alexa/node_modules/hap-node-client/HAPNodeJSClient.js:142:8)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at listOnTimeout (internal/timers.js:535:17)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]:     at processTimers (internal/timers.js:479:7)
Apr 21 22:01:18 raspberrypi-3BPlus homebridge[967]: [2019-4-21 22:01:18] Got SIGTERM, shutting down Homebridge...
```

**Comment by NorthernMan54** [comment]
@DJay-X To dig further into this, can you share an accessory dump of you homebridge instance?  I want to run it thru the device parser to determine why it is crashing.

https://github.com/NorthernMan54/homebridge-alexa#homebridge-accessory-dump

**Comment by DJay-X** [comment]
@NorthernMan54 Thanks for your fast reply.
Here you go with the accessory dump.
Thanks again.

[accessorydump.txt](https://github.com/NorthernMan54/homebridge-alexa/files/3103104/accessorydump.txt)

**Comment by NorthernMan54** [comment]
Tks for the log, helped find the issue really quickly.  This should be resolved with release v0.4.11

What had tripped up the code, was my initial support for HomeKit Television ( On/Off and Volume ) and the Harmony Plugins support for Television.  All should be good.

**Comment by DJay-X** [comment]
> Tks for the log, helped find the issue really quickly. This should be resolved with release v0.4.11
> 

I can confirm, with v.0.4.11 homebridge is running great again.
Thanks a lot for your great work and sharing.
